.circleci/config.yml

```diff
@@ -6,23 +6,48 @@ defaults: &defaults
 version: 2
 jobs:
   test-cypress:
-    docker:
-      - image: cypress/base:12.16.1
-        environment:
-          ## this enables colors in the output
-          TERM: xterm
+    machine:
+      image: ubuntu-1604:201903-01
```

Comment:
That's old...

Reply:
@sparkiegeek this looks like the most recent version available on CircleCI (https://circleci.com/docs/2.0/configuration-reference/#available-machine-images). Is that going to be an issue?

Reply:
Unfortunately the MAAS db initialisation script requires 18.04, and as we can't run the snap in a docker image, we're not left with many options. @sparkiegeek @albertodonato I suspect the issue is that we need the version of postgresql in 18.04? Would adding a PPA with a backport for 16.04 work?

Reply:
Yes, it should. MAAS currently requires PostgreSQL 10 at least, so you could probably add the appropriate repo from https://wiki.postgresql.org/wiki/Apt